### PR TITLE
feat: Upload Menu for Mobile

### PR DESCRIPTION
**Pull request by Crash--**
Capture d’écran 2019-07-01 à 08 52 48

- Not the right icons yet, we need an UI update
- Not added on the web version yet

### CODE REVIEW

File: packages/cozy-procedures/src/locales/en.json [hunk 1]
```diff
@@ -113,6 +113,14 @@
       "payslip": "Last payslip",
       "bank_identity": "Bank identity",
       "address_certificate": "Certificate of address"
+    },
+    "upload": {
+      "from_other_service": "From other service",
```
Comment (y-lohse) [inline packages/cozy-procedures/src/locales/en.json]:
```suggestion
      "from_other_service": "From another service",
```

File: packages/cozy-procedures/src/locales/en.json [hunk 2]
```diff
@@ -113,6 +113,14 @@
...
+      "from_drive": "From My Drive",
+      "from_my_mobile": "From my smartphone",
+      "from_my_device_scan": "Scan / Import from my smartphone",
+      "from_my_computer": "From my computer",
+      "soon_available": "(soon available)"
```
Comment (y-lohse) [inline packages/cozy-procedures/src/locales/en.json]:
```suggestion
      "soon_available": "(available soon)"
```

File: packages/cozy-procedures/src/components/documents/EmptyDocumentHolder.jsx [hunk 3]
```diff
@@ -1,67 +1,142 @@
...
+            theme="ghost"
+            extension="full"
+            align="left"
+            onClick={() => this.showMenu()}
+          >
+            <Icon icon="plus" size={16} className="u-mr-1 u-pa-half" />
+            <span>{t('documents.import')}</span>
+          </Button>
+          {this.state.menuDisplayed && (
+            <ActionMenu onClose={() => this.hideMenu()}>
```
Comment (cedricmessiant) [inline packages/cozy-procedures/src/components/documents/EmptyDocumentHolder.jsx]: I would prefer the action menu to be in a separate component

Comment (Crash--) [reply packages/cozy-procedures/src/components/documents/EmptyDocumentHolder.jsx]: Thanks! 

In the next PR we'll a WebMenu and a MobileActionMenu ;)

**Pull request merged**